com.unity.perception/Runtime/GroundTruth/Labeling/LabelManager.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System.Collections.Generic;
+using UnityEngine.Perception.Randomization.Randomizers;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    /// <summary>
+    /// Manages the registration of <see cref="Labeling"/> components
+    /// </summary>
+    public class LabelManager
```

Comment:
I've kicked off a performance test run on this branch. We should compare the results to the lastest on master to look for significant performance changes.

Reply:
https://yamato.cds.internal.unity3d.com/jobs/907-com.unity.perception/tree/remove-entities-dependency/.yamato%252Fupm-ci-full.yml%2523all_tests_nightly_perf/5697974/job